### DISCUSSION TITLE
Infill pattern questions

### DISCUSSION THREAD

**Discussion by balloni55** [supermerill/SuperSlicer#1269] [Q&A] [answered]
Version 2.3.56
Hello
i like the infill patterns for top layer very much and use them often.

1. in sliced preview as well as in printed parts there are square outlines visible if i use "Hilbert curve"
 is there any setting to suppress this lines like a endless pattern?

2. with ironing post-process it is possible to use it only for the "top most layer", how can i achieve this for the innfill pattern?
![square](https://user-images.githubusercontent.com/73615599/121007341-37305880-c792-11eb-9700-a72fc4f2278b.jpg)

**Comment by supermerill** [comment, marked as answer]
> how can i achieve this for the innfill pattern?

use a range modifer
* right-clic on your object
* height range modifier
* then on the right panel clic on it
*  enter the layer height where you want your ironing
* clic on the range modifier cog -> add setting -> infill -> top pattern
* change top pattern to ironing

**Reply by supermerill** [reply]
for the hilbert curve, it's a fractal, so it's not possible. Maybe by overextruding a bit...

**Comment by balloni55** [comment]
thanks for your detailed advice, it work perfect :+1:

over-extrusion unfortunately does not achieve any improvement
